front/app/components/admin/InternalComments/ParentCommentForm.tsx

```diff
@@ -243,6 +243,9 @@ const ParentCommentForm = ({
 
     return null;
   };
+  const textAreaPlaceholder = formatMessage(
+    commentsMessages.notVisibleToUsersPlaceholder
+  );
```

Comment:
Could put it directly in the prop, makes code slightly cleaner :-)

Reply:
🤔 In my opinion, I think it makes the JSX harder to read whereas defining it is straight forward

Reply:
Matter of preference and what you're used to then, I guess 🙂